pkg/p2p/peer/handshake.go

```diff
@@ -12,20 +12,20 @@ import (
 )
 
 // Handshake with another peer.
-func (p *Writer) Handshake() error {
-	if err := p.writeLocalMsgVersion(p.gossip); err != nil {
+func (w *Writer) Handshake() error {
```

Comment:
this was renamed to be consistent with other `(w *Writer)`

Reply:
OK